build/pipelines/templates/build-app-internal.yaml

```
@@ -29,7 +29,7 @@ jobs:
       downloadDirectory: $(Build.SourcesDirectory)
       vstsFeed: WindowsApps
       vstsFeedPackage: calculator-internals
-      vstsPackageVersion: 0.0.31
+      vstsPackageVersion: 0.0.32
```

I just checked your change is in version 0.0.33

```suggestion
      vstsPackageVersion: 0.0.33
```

---

You are right! I must have resolved the merge conflicts wrong. Thanks!